### DISCUSSION TITLE
Term obsoletion: the do and don't

### DISCUSSION THREAD

**Discussion by sabrinatoro** [jamesaoverton/obook#177] [Tutorial requests] [answered]
It would be good to follow-up on term obsoletion: we already had a tutorial on how to obsolete terms, but it would be good to talk about
- when should a term be obsoleted (e.g the splitting terms)?
- can we "un-obsolete" a term? 
- What are the consequences of term obsoletion/un-obsoletion on data annotation using ontologies?

**Comment by matentzn** [comment]
Hmm.. Not sure I would want to do this soon again.. Obsoletion is a very niche problem for most ontologies other than Mondo and Uberon - I would prefer if someone would take the time and update the docs around obsoletion to just add 2 paragraphs along these lines, and next time we get back to "the term obsoletion tutorial (2023)" we just make sure we cover these issues as well. What do you think?

**Comment by nicolevasilevsky** [comment, marked as answer]
that sounds good to me.
